string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,13 @@
+/* exported numVowels */
+function numVowels(string) {
+  var currentCount = 0;
+  if (string.length === 0) {
+    return 0;
+  }
```

Comment:
This guard isn't necessary, since if the loop executes and there is no length to the string, you would still be returning `0` at the end of the function